npm/npm_import.bzl

```diff
@@ -30,6 +30,7 @@ load("//npm/private:npm_import.bzl", _npm_import_lib = "npm_import", _npm_import
 load("//npm/private:versions.bzl", "PNPM_VERSIONS")
 load("//npm/private:utils.bzl", _utils = "utils")
 load("//npm/private:npm_translate_lock.bzl", _npm_translate_lock = "npm_translate_lock")
+load("@aspect_bazel_lib//lib:repositories.bzl", _register_copy_to_directory_toolchains = "register_copy_to_directory_toolchains")
```

Comment:
nit: IIUC something changed, and you no longer have to rename these to underscore on import - imported symbols aren't automatically treated as exports by stardoc

Reply:
in this case `register_copy_to_directory_toolchains` conflicts with the attribute name so needs the rename but that is good to know